app/unitPages.rb

```diff
@@ -653,6 +718,7 @@ def getNavByID(navBar, navID)
 def deleteNavByID(navBar, navID)
   return navBar.map { |nav|
     nav['id'].to_s == navID.to_s ? nil
+      # Check for 'folder'    (You would never want to delete a 'fixed_folder')
```

Comment:
Ah, there's the answer to my question. "fixed_folder" is the other kind of folder. Ok.